obs-script/Files/settings/Logo.json

```diff
@@ -0,0 +1,3 @@
+{
+    "file": "C:/Users/fabio/fabio/Google Drive/RLBot/ArtAssets/Wintertide/Overlay/wintertide ingame overlay wins counters bo3 0.png"
+}
```

Comment:
Hardcoded path

Reply:
this path doesn't matter
https://github.com/RLBot/AutoLeaguePlay/blob/9fefec9e29b38a867abb847a98e51a9e94ef554e/obs-script/Files/settings/Boost-0.json#L2
It always had a path and always worked. That path gets replaced when it sets up the scene, but it needs the file: stuff to be able to create the image, even if nonexistant

Reply:
Oh, I see. Is it something we should put in the .gitignore then?